homeassistant/components/alarm_control_panel/services.yaml

```diff
@@ -60,6 +60,16 @@ envisalink_alarm_keypress:
       description: 'String to send to the alarm panel (1-6 characters).'
       example: '*71'
 
+alarmdecoder_alarm_keypress:
```

Comment:
Please move this to the services.yaml file in the alarmdecoder integration package.